src/cmp-ui.test.ts

```diff
@@ -1,15 +1,15 @@
 import { setupMessageHandlers, canShow } from './cmp-ui';
 import { CMP_DOMAIN, CMP_READY_MSG, CMP_CLOSE_MSG } from './config';
 import {
-    readGuCookie as _readGuCookie,
+    // readGuCookie as _readGuCookie,
```

Comment:
Do these need to be commented out here, or is it better to just remove them>

Reply:
I commented these out as the exclusion of `readGuCookie()` from the `canShow` check in `cmp-ui.ts` is temporary, whilst we're not handling consent information for the PECR purposes. I felt leaving them here but commented out gave us an indication of what we intend to be doing once we start handling consent information for the PECR purposes.